src/Util/Autoload/ClassLoaderMethod/Exception/SignatureCheckFailed.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types = 1);
```

Comment:
Good point; except to use `Roave\Signature` library, we need 7.0+ anyway, so I still need to bump the min version here (see #221)